jnigen/lib/src/bindings/dart_generator.dart

```diff
@@ -63,6 +73,31 @@ String _newLine({int depth = 0}) {
   return '\n${'  ' * depth}';
 }
 
+/// Merges two maps by adding all the values of one to the other.
```

Comment:
Nit: you're making a new outer map rather than adding the values of one to the other.

Reply:
I will improve the wordings, these are "multimaps", i.e. maps with a List as their values, I'm merging these lists, and I'm not creating new lists, I reuse the larger one to get a logarithmic complexity instead of a quadratic one.